Tasks/XamariniOS/xamarinios.ts

```diff
@@ -147,9 +148,14 @@ nugetRunner.exec()
                                             .then(function (code) {
                                                 tl.exit(code);
                                             })
+                                    } else {
+                                        tl.exit(0);
```

Comment:
tl.exit(0) doesn't actually stop execution and I thought this was obsolete and shouldn't be used. Instead I think we should use setTaskResult(). I assume this is needed for the L0 tests.